### DISCUSSION TITLE
Batch order to Europe / Germany in 3 / 2026 - New units available

### DISCUSSION THREAD

**Discussion by oleo65** [rbaron/b-parasite#173] [General]
🎉 Update 3/2026: New units arrived and some are still available. Please see https://github.com/rbaron/b-parasite/discussions/173#discussioncomment-9312938 

Original Posting:

I am planning to order a new batch of parasites v2.0.0 in January 2024.

I would like to give you the opportunity to join in so that we can achieve better cost per unit due to very high on time cost per order.

As a service you get the fully assembled parasite including the soldered battery clip. Optionally I can flash the chip with the firmware of choice for you.

The final cost for unflashed parasites will probably be between 14-19 € per unit, depending on the batch size we can achieve. 

Please leave a message here or drop me an email if you are interested until January 21, 2024. 📅

**Comment by hubertgruber** [comment]
I am interested!

**Reply by oleo65** [reply]
Great, just let me know via E-Mail in how many pieces you are interested in.

**Comment by Pferdebockwurst** [comment]
I'm interested too! Can I join even if it's February already?

**Reply by oleo65** [reply]
Yes you can, order is already on the way but I have some units more to spare.

**Comment by mweedon** [comment]
Is it still possible to get 5 or so of these to the UK or have I woken up too late??@

**Reply by oleo65** [reply]
Yes you are lucky, please drop me an email to discuss details. 😊

**Comment by GodnessHimself** [comment]
Hello,
I have just become aware of the project.
I would also be interested in some sensors.
Have you already ordered them? :(

**Reply by oleo65** [reply]
I have a few pieces left from the last order. Please contact me by email for details.

**Comment by oleo65** [comment]
The current batch is now spread out all over Europe. It's really amazing to see how many people are interested in this awesome project.

I am happy to support this by offering to order a new batch if there is enough interest.

If you are interested, please leave a comment for now including how many pieces you would like to receive. The factor of scale is quite significant when ordering, so the more the better. 😊

**Reply by pikkmarg** [reply]
What did the unit cost come down to for the last batch? I am probably looking for 4 units.

**Reply by oleo65** [reply]
I expect them to be between 15-17 € per piece. The prices of components especially the MC are fluctuating so I cannot give an exact estimation in advance.

**Reply by pikkmarg** [reply]
Cool! I understand the nature of the order so its all good. Let me know if you need anything else  :)

**Reply by fsalfner-dev** [reply]
I would be interested, too, when do you think you will place the order?

**Reply by oleo65** [reply]
That depends on the interest. If more people are interested I suppose I would place the order within the next 4-8 weeks.

My target would be a batch of at least 50 pieces, and I would like to reach at least 50% pre orders of the batch size before ordering.

So please leave a number of units in which you are interested.

**Reply by Nej4218** [reply]
I'm be very interested in some time  (need to finish other automation first), but building a plant wall and ideally I'd like one in each planter (I have 272 planters on the wall), can't afford to buy that many in bulk at once right now, but hope we can figure something out when it comes to ordering. Sidenote, would there be any interference or other stuff I have to worry about having that many that close together? 

Regardless I'll shoot you an email and hopefully we can discuss further (I'm from Denmark btw)

**Comment by mweedon** [comment]
I'll take another 15 (zigbee version) please!


Regards


Mark

**Comment by PineappleEmperor** [comment]
I'd go for 20-30 Zigbee ones please (depending on price) :)

**Comment by kzyapkov** [comment]
Is this still open? If so, count me in with 10!

**Comment by jsk-sct** [comment]
Hi
I'm interested in being a part of the next batch. I would like 6 or 7 units of the zigbee version. Do you want a mail or is a comment here sufficient?

**Comment by Copyrighter** [comment]
@oleo65 
Is it possible to join? I will be interested in next batch.
Can You assist with explaining where did You ordered PCB? Was it completely soldered?
thank You

**Comment by oleo65** [comment]
Hey everyone,

I have great news for all of you who are interested in some more parasites. ~~They are currently produced and I expect them to be ready to be shipped to you within the next 2-3 weeks (fingers crossed for fast custom clearance...).~~ They arrived and are ready to ship. 🥳

Please, if you are interested drop me an **email** (in my Github profile). That is the only way I can provide you with further details without disclosing to much private information about you on GitHub. 😃 

**Please add:**

- how many devices you would like to receive, 
- what firmware (ZigBee / Bluetooth / None you do it yourself)
- to which country they will be shipped to 

Thank you!

**Reply by maba64** [reply]
Hi
im intrested in 2 pcs with Bluetooth
br
Markus

**Reply by oleo65** [reply]
Please see https://github.com/rbaron/b-parasite/discussions/173#discussioncomment-9312938 and get in touch by E-mail.

**Comment by Copyrighter** [comment]
Hi,
I would like to order one item for testing ( or 3 ... for any case ). Zigbee version.
At the end, I will need more than 30 items.
Best Regards,
Alexey

**Reply by oleo65** [reply]
Please write me an **email** with your details and not disclosing your private data on this public thread. 👼

**Comment by eliasel** [comment]
This is great stuff! 
I have sent you an email, hoping there are still some left :)
